packages/jupyter-ai/jupyter_ai/personas/persona_manager.py

```diff
@@ -52,15 +53,13 @@ class PersonaManager(LoggingConfigurable):
     type for type checkers.
     """
 
-    # TODO: the Persona classes from entry points should be stored as a class
-    # attribute, since they will not change at runtime.
-    # That should be injected into this instance attribute when personas defined
-    # under `.jupyter` are loaded.
-    _persona_classes: list[type[BasePersona]] | None = None
+    # We treat this as a class attribute so that we only have to load them once
+    _ep_persona_classes: list[dict] | None = None
```

Comment:
Class attributes require a special annotation:

```suggestion
    _ep_persona_classes: ClassVar[list[dict] | None] = None
```

This can be imported from `typing`.